.github/workflows/nightly_tests.yml

```diff
@@ -99,7 +96,7 @@ jobs:
       if: ${{ always() }}
       uses: actions/upload-artifact@v2
       with:
-        name: ctrd-logs
+        name: ${{ github.job }}-${{ matrix.vhive_args }}-ctrd-logs
```

Comment:
Is changing the name here enough?
Value of `CTRDLOGDIR` inside `start_onenode_vhive_cluster.sh` also will have to be changed, right?

Reply:
the folder name can be the same because one runner processes one job at a time. However, each folder must be uploaded as a separate archive to be available for downloading.